main.lua

```diff
@@ -465,7 +473,7 @@ function LSPClient:handleResponseResult(method, result)
             end
             table.insert(symbolLabels, string.format("[%s]\t%s", SYMBOLKINDS[sym.kind], sym.name))
```

Comment:
This should fix the alignment:
```suggestion
            table.insert(symbolLabels, string.format("%-15s %s", "["..SYMBOLKINDS[sym.kind].."]", sym.name))
```

Reply:
I thought those labels were directly from the LSP. I replaced the tab with the size that it occupies, but these are fewer operations. I will use this instead.